.pre-commit-config.yaml

```diff
@@ -1,19 +0,0 @@
-files: '(screenpy_pyotp|tests)/.*'
-fail_fast: false
-repos:
-  - repo: https://github.com/psf/black
-    rev: 24.1.1
-    hooks:
-      - id: black
-        language_version: python3.12
-  - repo: https://github.com/astral-sh/ruff-pre-commit
-    # Ruff version.
-    rev: v0.2.0
-    hooks:
-      - id: ruff
-  - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v1.8.0
-    hooks:
-      - id: mypy
-        language_version: python3.12
-        additional_dependencies: ["pyotp>=2.3.0"]
```

Comment:
Since we are removing the pre-commit, we should update the readme and remove the step that says to run `pre-commit install`.   I updated the cookie cutter. `cruft update` should do the trick.